presto-main/src/main/java/com/facebook/presto/sql/planner/optimizations/AddExchanges.java

```diff
@@ -1251,6 +1277,22 @@ private Partitioning createPartitioning(List<Symbol> partitioningColumns)
             PartitioningHandle partitioningHandle = metadata.getPartitioningHandleForExchange(session, partitioningProviderCatalog, hashPartitionCount, partitioningTypes);
             return Partitioning.create(partitioningHandle, partitioningColumns);
         }
+
+        private Scope selectExchangeScopeForPartitionedRemoteExchange(PlanNode exchangeSource)
+        {
+            if (exchangeSource.getOutputSymbols().isEmpty()) {
+                // materializing 0 columns input is not supported
```

Comment:
curious: when does this going to happen ?  -- maybe I can again set a breakpoint .. :)

Reply:
It was related to some weird JOIN + COUNT(*) cases. I don't remember exactly.